docs/usage.rst

```diff
@@ -322,12 +326,15 @@ A Docker container can be created using the following command:
 .. code-block:: bash
 
    docker run --rm -it \
-      -v /dset/derivatives/fmriprep:/fmriprep:ro \
+      -v /path/to/fmriprep_dir:/fmriprep:ro \
       -v /tmp/wkdir:/work:rw \
-      -v /dset/derivatives/xcp_d:/out:rw \
-      pennlinc/xcp_d:latest \
-      /fmriprep /out participant \
-      --file-format cifti --despike --head_radius 40 -w /work --smoothing 6
+      -v /path/to/output_dir:/out:rw \
+      pennlinc/xcp_d:<version> \
+      /path/to/fmriprep_dir \
+      /path/to/output_dir \
```

Comment:
`/path/to/fmriprep_dir` is mounted to `/fmriprep` in the container, so you need to use `/fmriprep` here. Same for `/path/to/output_dir`.